bitnami/redis/templates/redis-master-statefulset.yaml

```diff
@@ -396,6 +402,7 @@ spec:
             - name: REDIS_EXPORTER_TLS_CLIENT_CERT_FILE
               value: {{ template "redis.tlsCert" . }}
             {{- end }}
+          {{- if or .Values.usePasswordFile .Values.tls.enabled }}
```

Comment:
Why is this `if` needed?

Reply:
See screenshot for this and other examples